src/account/PasskeyAccount.sol

```diff
@@ -0,0 +1,260 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import { Base64Url } from "../utils/Base64Url.sol";
+import { IEntryPoint, UserOperation } from "@aa/interfaces/IEntryPoint.sol";
+import { IPasskeyAccount, Passkey } from "../interfaces/IPasskeyAccount.sol";
+import { P256Verifier } from "../utils/P256Verifier.sol";
+import { SimpleAccount } from "@aa/samples/SimpleAccount.sol";
+
+contract PasskeyAccount is SimpleAccount, IPasskeyAccount {
+    P256Verifier public immutable p256Verifier;
+    Passkey private passkey;
```

Comment:
This should allow to read, we could make it public or add another external function `getPasskey`.